README.md

```diff
@@ -1,6 +1,34 @@
 # virtual_shake_robot_pybullet
 Virtual shake robot v2 based on PyBullet for earthquake studies
 
-Dependencies : 
+# Project  Overview:
+The Virtual Shake Robot (VSR) project is a sophisticated simulation tool designed to study the dynamics of Precariously Balanced Rocks (PBRs) during overturning and large-displacement processes. Utilizing the PyBullet physics engine, the VSR offers an innovative approach to understanding seismic hazards, rockfall prediction, and the fate of toppled PBRs with realistic material properties and terrains. This project is developed with the integration of advanced robotics and machine learning technologies, aiming to automate rock mapping and analysis effectively.
 
+# Simulation Software:
 
+## PyBullet ##
+It is highly recommended to use PyBullet Python bindings for improved support for robotics, reinforcement learning and VR. Use pip install pybullet and checkout the [PyBullet Quickstart Guide](https://docs.google.com/document/d/10sXEhzFRSnvFcl3XxNGhnD4N2SedqwdAvK3dsihxVUA/edit#heading=h.2ye70wns7io3).
```

Comment:
We use PyBullet Python bindings for improved support for robotics, reinforcement learning and VR. Use pip install pybullet and checkout the